<launch>

  <node pkg="map_compression" type="genWeightVector" name="genWeightVector" output="screen">
  
  <param name="loadMapName" value="/home/yh/mapModel/2018/08.03/wholeMap_yq_scored.vtk"/>
  <param name="saveWeightName" value="/home/yh/mapModel/2018/08.03/weightVector.txt"/>

  </node>
</launch>
